sdk/storage/azure-storage-common/inc/azure/storage/common/paged_response.hpp

```diff
@@ -30,6 +30,7 @@ template <class Derived> class PagedResponse {
       m_hasMorePages = false;
       return;
     }
+
```

Comment:
nit: Revert this unnecessary change from here, to keep history/blame clean, especially since we plan to remove this file later.